### PR TITLE
WIP: Resolve SEARCH-1546 "Feature/search 1546 docker master slave config"

### PR DESCRIPTION
**Pull request by sridharvellingiri**
Bamboo build needs to be fixed

### CODE REVIEW

File: packaging/src/docker/6.x/nginx.conf [hunk 1]
```diff
@@ -0,0 +1,15 @@
+server {
```
Comment (michaelsuzukisagi) [inline packaging/src/docker/6.x/nginx.conf]: Comment on why and what is the purpose of the nginx

**Comment by michaelsuzukisagi** [comment]
I cant see how the archive collection is replicated in master/slave setup

File: packaging/src/docker/6.x/nginx.conf (hunk 1 above)
Comment (sridharvellingiri) [reply packaging/src/docker/6.x/nginx.conf]: nginx used here as a load balancer for single or multiple slaves for all quires from Alfresco. This is optional when we deploy along with slave config.

**Comment by sridharvellingiri** [comment]
> I cant see how the archive collection is replicated in master/slave setup

We are passing ${solr.core.name} value in the script file for slave setup which will overwrite with correct core name for url in the solrcore.property file. Based on ${solr.core.name} value it will replicate all available cores from the Master.

File: packaging/src/docker/6.x/nginx.conf (hunk 1 above)
Comment (michaelsuzukisagi) [reply packaging/src/docker/6.x/nginx.conf]: Please add the comment in the code.

**Pull request closed without merging**